logstash/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG - Logstash
 
+##
```

Comment:
```suggestion
## 1.2.0
```


or will this autopopulate?

Reply:
Nah I don't think it does - thanks for catching.